ci-operator/step-registry/ipi/conf/gcp/compute-node-type/ipi-conf-gcp-compute-node-type-ref.yaml

```diff
@@ -0,0 +1,22 @@
+ref:
+  as: ipi-conf-gcp-compute-node-type
+  from_image:
+    namespace: ocp
+    name: "4.16"
+    tag: upi-installer
+  commands: ipi-conf-gcp-compute-node-type-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: COMPUTE_DISK_TYPE
```

Comment:
`COMPUTE_DISK_TYPE` is already supported by `ipi-conf-gcp-osdisk-disktype-compute`
`COMPUTE_NODE_TYPE` is already supported by `ipi-conf-gcp`

Sounds like this is a duplicated step.

Reply:
@dfajmon Could you please try to reuse the `ipi-conf-gcp` and `ipi-conf-gcp-osdisk-disktype-compute` to avoid duplicated steps? Thanks!

Reply:
updated but we need the `storage-conf-storageclass-pre-set-default-gcp-hyperdisk` because we are setting the `storageClassState: Unmanaged`